### ISSUE TITLE
Initial jupyter notebook segmentation looks different from GUI; then self-corrects on second run

### ISSUE THREAD

**Issue by mssher07** [MouseLand/cellpose#639]
Hello--first, have to say cellpose 2.0 is amazing, and the ability to run the "model zoo" to find an optimal solution to a specific problem is a huge improvement compared to prior segmentation solutions.

Through the GUI, I identified model 'CPx' as working for my segmentation problem out-of-the-box. However, when I went run the same model in a Python / jupyter lab environment (for batch run purposes), I am seeing a very different output. I am not sure how to reconcile the differences or where the issue is.

The jupyter lab snippet is here:
```
# RUN CELLPOSE
from cellpose import models, io

master_diam=90

# DEFINE CELLPOSE MODEL
model = models.CellposeModel(model_type='CPx',diam_mean=master_diam)
imgs = [io.imread(filename) for filename in files]
masks, flows, styles = model.eval(imgs, diameter=master_diam, channels=channels)

diams=[master_diam]
io.masks_flows_to_seg(imgs, masks, flows, diams, files, channels)
io.save_to_png(imgs, masks, flows, files)
```

cp segmentation masks
![image](https://user-images.githubusercontent.com/60016050/211625803-89329a9a-8122-4840-92af-364a8d3c2172.png)

expected (correct) output from GUI run segmentation
![image](https://user-images.githubusercontent.com/60016050/212167843-189513e9-5be3-4b6e-baf7-0116d15964f0.png)

EDIT 1. OK now it's working. No idea what I changed.

EDIT 2. OK figured it out, this seems to happen _only for the first run_ in jupyter lab. Running the exact same code a second time produces the correct output. Have seen this both with 'CPx' and a pretrained model; initial output is totally different, then somehow self corrects for the second run.

**Comment by carsen-stringer** [comment]
this is indeed strange, can you please turn on the logger in jupyter and send the output?

turn on logger with `logger = io.logger_setup()`

**Comment by carsen-stringer** [comment]
closing due to inactivity, please let us know if you have any feedback
